README.md

```diff
@@ -1,112 +1,93 @@
+Pace is an implementation of the FV3GFS / SHiELD atmospheric model developed by NOAA/GFDL using the GT4Py domain-specific language in Python. The model can be run on a laptop using Python-based backend or on thousands of heterogeneous compute nodes of a large supercomputer.
 
-If you are visiting for AMS 2022, we recommend you go to `driver/README.md`.
+The top level directory contains the FV3 dynamical core (fv3core), the GFS physics package (fv3gfs-physics), and infrastructure utilities (pace-util).
 
-**WARNING** This repo is under active development and relies on code and data that is not publicly available at this point.
+**WARNING** This repo is under active development and supported features and procedures can change rapidly and without notice.
 
 This git repository is laid out as a mono-repo, containing multiple independent projects. Because of this, it is important not to introduce unintended dependencies between projects. The graph below indicates a project depends on another by an arrow pointing from the parent project to its dependency. For example, the tests for fv3core should be able to run with only files contained under the fv3core and util projects, and should not access any files in the driver or physics packages. Only the top-level tests in Pace are allowed to read all files.
 
 ![Graph of interdependencies of Pace modules, generated from dependences.dot](./dependencies.svg)
 
-## Dynamical core tests
+## Building the docker container
 
-Before the top-level build, make sure you have configured the authentication with user credientials and configured Docker with the following commands:
-```shell
-$ gcloud auth login
-$ gcloud auth configure-docker
-```
+While it is possible to install and build pace bare-metal, the easiest setup is to use a Docker container for testing and developing pace.
 
-You will also need to update the git submodules are cloned and at the correct version:
+First, you will need to update the git submodules so that any dependencies are cloned and at the correct version:
 ```shell
-$ git submodule update --init --recursive
+git submodule update --init --recursive
 ```
 
-To run dynamical core tests, first get the test data from inside `fv3core` or `fv3gfs-physics` folder, then build `fv3gfs-integration` docker image at the top level.
-
+Then build the `pace` docker image at the top level.
 ```shell
-$ cd fv3core
-$ make get_test_data
-$ cd ../
-$ make build
+make build
 ```
 
-For serial tests (these take a bit of time), there are two options:
+## Downloading test data
 
-(1) To enter the container and run the dynamical core serial tests (main and savepoint tests):
+The unit and regression tests of pace require data generated from the Fortran reference implementation which has to be downloaded from a Google Cloud Platform storage bucket. Since the bucket is setup as "requester pays", you need a valid GCP account to download the test data.
```

Comment:
> Should you also include the FTP instructions in this section?

This was merged in another PR. I will merge with upstream main and then it will be in.